packages/zact/client.ts

```diff
@@ -1,5 +1,3 @@
-"use client";
```

Comment:
Decent chance this won't work without this

Reply:
I'm pretty confident about this. 
Here you are in a library. 
Library code is built and bundled without knowing anything about RSC boundaries. 
This should be at component level in your next app :

```ts
'use client'
import { useZact } from 'zact/client';
```
cf https://github.com/reactjs/rfcs/blob/main/text/0227-server-module-conventions.md

Reply:
Well true, we should use hook in client component anyway, user will have to specify the `use client` instead, so the directive is not really necessary in here.

However, it doesn't hurt to have the `use client` directive here tho, to tell React that this is one of the client entry, it's safer than nothing.

Reply:
The issue is, tsup used here to bundle does not honor 'use client' placement at the top of the file.
So that's why i propose to remove it, since it's  packaged correctly